torch/_inductor/graph.py

```diff
@@ -1864,6 +1864,7 @@ def get_output_names(self) -> List[str]:
     def is_unspec_arg(self, name: str) -> bool:
         # dynamo wraps unspec variable as 0d CPU tensor,
         # need to convert to scalar during codegen (triton only)
+        return False  # FIXME
```

Comment:
Looking for ideas on how to tackle this; I don't really understand why unspec variables need to be represented like so

Reply:
This was introduced in #87329; cc @jansel @yanboliang

Reply:
This is because an unspecialized scalar is passed as a single-element cpu tensor input after dynamo. Should be ok to always return False for CPU-Triton.